changefeedtrigger/changefeedSample.py

```diff
@@ -0,0 +1,81 @@
+from .settings import *
+from azure.storage.blob.changefeed import ChangeFeedClient
+from azure.storage.blob import BlobServiceClient
+import azure.functions as func
+import logging
+from azure.core.exceptions import ResourceNotFoundError
+
+
+def main(mytimer: func.TimerRequest) -> None:
+
+    # create blob service client to access connection string
+    blob_service_client = BlobServiceClient.from_connection_string(CONNECTION_STRING)
+    change_feed_client = ChangeFeedClient.from_connection_string(CONNECTION_STRING)
+
+    # create or get container to store cursor
+    container_name = "cursorstoragecontainer"
+    try:
+        token_container_client = blob_service_client.create_container(container_name)
+    except:
+        token_container_client = blob_service_client.get_container_client(container_name)
+
+    # call to method
+    events = get_events(token_container_client, change_feed_client)
+    events = filter_events(events)
+ 
+    # logging filtered event output
+    if len(events) == 0:
+        logging.info("\nNo events matching filters on current page...")
+    else:
+        logging.info("\nRecents event(s) matching filters on current page:")
+        count = 1
+        for e in events:
+            logging.info("%d: %s\n" % (count, e))
+            count += 1
+
+
+def get_events(token_client, cf_client):
+    # this method uses the continuation token to iterate on events and updates the token
+
+    # check if continuation token already exists
+    blob_name = "cursorBlob"
+    blob_client = token_client.get_blob_client(blob_name)
+    try:
+        blob_client.get_blob_properties
+        cursor = blob_client.download_blob().readall()
+
+        # initiate the changefeed with cursor
+        change_feed = cf_client.list_changes(results_per_page=EVENTS_PER_PAGE_QUANTITY).by_page(continuation_token=eval(cursor))
+
+    # if token does not exist upload token as empty string placeholder as blob and initiate changefeed without cursor
+    except ResourceNotFoundError: 
+        change_feed = cf_client.list_changes(results_per_page=EVENTS_PER_PAGE_QUANTITY).by_page()
+        
+    # iterate through change feed
+    change_feed_page = next(change_feed)
```

Comment:
>change_feed_page [](start = 4, length = 16)

only the first page will be printed out?

Reply:
yes, but there are X amount of events per page. I think right now it is set to 50 events per page. This line should be filtering the first 50 events in the changefeed log.

Reply:
isn't it expected to be a list of events within a time range?

Reply:
it's not filtering within a time range, just by container and blob name. the use of the page makes it so that it would print out new events that occur, once it goes through all of the changefeed log. so when it gets to the end of all of the events it doesnt do anything, but then when new events occur it goes through those new events. so its not that it's events within a time range, but that it is tracking new events that are occurring.